Integrations/integration-RsaNetWitnessPacketsAndLogs.yml

```diff
@@ -219,6 +224,10 @@ script:
         var responseType = resType ? ResponseTypes[resType] : 'application/json';
         var fullUrl = url + path + parametersUrl + '&force-content-type=' + responseType;
 
+        if (params.expiry) {
```

Comment:
Do we want expire as param? If yes then all the commands in this instance will have the same expire.